addons/main/CfgAIProfile.hpp

```diff
@@ -0,0 +1,12 @@
+class LAMBS_CfgAIProfiles {
+    class Default {
+        #ifdef ISDEV
+        allowDemoNumberAsBool = 1;
```

Comment:
All of the profile stuff is in the main component, wouldn't it be better to have the dev values in danger and the actual prod default values in main?

Reply:
I would split it into the modules where it is used

Reply:
We do need then at least in the wiki some entry with all config entries. It is confusing to look at the code and only see a fraction of possible entries. (Or we have to tell people to use the config viewer)

Reply:
originally I planned to build a tool that shows and allows you to create and modify profiles in the editor in the release after the profiles where introduced. but writing a small export for the wiki is also fairly easy